Develop

happy-land opened the pull request.

annasilina committed:
first steps at profile page form CSS

trashmarket committed:
select 0.0.1 done

trashmarket committed:
calendare #0.0.1

happy-land committed:
Merge pull request #17 from happy-land/ruslan-main-page

Ruslan main page

annasilina committed:
added map page

annasilina committed:
added apiKey for map element

Dumisel committed:
логика с реакциями

annasilina committed:
fixed apikey for map element

happy-land committed:
Merge branch 'develop' into lisa-emoji

happy-land committed:
Merge pull request #19 from happy-land/lisa-emoji

логика с реакциями

happy-land committed:
comments-block hotfix

happy-land committed:
Merge pull request #21 from happy-land/comments-block-hotfix

comments-block hotfix

happy-land committed:
Merge branch 'develop' into anna-map-page

happy-land committed:
Merge pull request #20 from happy-land/anna-map-page

Anna map page

annasilina committed:
added directory for Api class
fixed common types

MaximTutunov committed:
fix1 enter the commit message for your changes. Lines starting

MaximTutunov committed:
image restored

MaximTutunov committed:
fixed

annasilina committed:
added all api requests

annasilina committed:
added type for getCohortData return

trashmarket committed:
fix colendar whith close

annasilina committed:
added types for all Api methods
added setToken method - for token visibility inside Api
added types for all objects

annasilina committed:
deleted old api.ts

annasilina committed:
fixed map page: deleted map component, added useState for data at MapPage

annasilina committed:
Merge remote-tracking branch 'origin/anna-map-page' into anna-map-page

Dumisel committed:
DetailPage счетчик, черновик routes

trashmarket committed:
form #0.0.1

happy-land committed:
Merge pull request #23 from happy-land/admin-page

Admin page

happy-land committed:
conflicts resolved

happy-land committed:
Merge pull request #24 from happy-land/anna-api

Anna api

happy-land committed:
conflicts resolved

happy-land committed:
Merge pull request #25 from happy-land/lisa-detailpage-more

DetailPage счетчик, черновик routes

happy-land committed:
conflicts resolved

happy-land committed:
Merge pull request #26 from happy-land/nikita-form-page

Nikita form page

happy-land committed:
Merge pull request #27 from happy-land/anna-map-page

Anna map page

annasilina committed:
fixed constants - deleted extra types
fixed types.ts
added path for Link to Map Page at Home page
added htmlType for Button
deleted "react-yandex-maps" package

annasilina committed:
fixed Api methods
fixed baseUrl in api config
fixed proxy url in package json

annasilina committed:
deleted constants.tsx
fixed styles for map's avatars

happy-land committed:
Merge pull request #28 from happy-land/anna-fix

fixed constants - deleted extra types

Dumisel committed:
исправлены апи

happy-land committed:
main page fix draft

happy-land committed:
Merge branch 'develop' into lisa-review

happy-land committed:
Merge pull request #29 from happy-land/lisa-review

исправлены апи

Dumisel committed:
рутс + обновленные тестовые данные

Dumisel committed:
Merge branch 'lisa-review' of https://github.com/happy-land/visitki into lisa-review

Dumisel committed:
типы

happy-land committed:
in progress

happy-land committed:
Merge pull request #30 from happy-land/lisa-review

Lisa review

happy-land committed:
fixing

happy-land committed:
error

happy-land committed:
main fixed

happy-land committed:
Merge pull request #31 from happy-land/ruslan-main-fix

Ruslan main fix

happy-land committed:
Merge branch 'main' into develop

gennady-bars reviewed (commented):
Здравствуйте. (**Нужно развернуть общий комментарий ↓**)

К сожалению, работу придется отклонить без потери итерации, так как проект не запускается. Скрин ошибки https://disk.yandex.ru/i/9UviLA0LblU4NA

Еще заметил ошибки:


* в `Readme` нужно указать ссылку на бриф по заданию
* в `Readme` нужно добавить описание проекта, кто участники, какая это группа (номер команды) и тд

annasilina committed:
fixed css emoji module
added project info to README

annasilina committed:
added info about current functionality to README
fix config for Api
fixed types
fixed Map Page
changed request data method at admin components
fixed unused imports in components
fixed duplicate declaration in type TReaction

happy-land commented:
Информация по текущей реализации указана в 5 пункте Readme.
